docs/modules/ROOT/pages/maintenance_process.adoc

```diff
@@ -0,0 +1,18 @@
+= Maintenance Process
+
+== Overview
+This page should provide information about our maintenance process.
+
+== APPUiO Public Maintenance Process
```

Comment:
This sections belongs to the APPUiO docs under https://github.com/appuio/docs

Reply:
Hmm, had a look on https://github.com/appuio/docs and I'm not sure if it really fits there. In APPUiO docs we do have this main sections:
* User Documentation
* APPUiO Microservices Example
* Application Tutorials

The "User Documentation" would be the best fit, but it seems to me it is more a "self service" section, where a APPUiO customer could get information to run their service.

Where would it best fit in your opinion? Perhaps a paragraph in http://docs.appuio.ch/en/latest/appuio-specifics-public.html

So in my opinion it is a better fit in the general KB about VSHN. Maintenance is a service by VSHN. And if somebody wants to know what maintenance process we provide, then it should be on centralized place.
Please let me know what you think about that.

Reply:
We can leave the documentation in kb, but then a link from docs.appuio.ch to kb.vshn.net is needed, pointing to this page. http://docs.appuio.ch/en/latest/appuio-specifics-public.html would be a good place for it, yes.